README.md

```diff
@@ -86,12 +86,14 @@ must include the following information:
       * `nodejs`
       * `python`
       * `binary`
+      * `none` - used only in the case of pure UI extensions
```

Comment:
Isn't this technically 'javascript'?

I'm guessing the distinction here is that the gateway doesn't actually execute that code, but "none" feels a bit strange.

Reply:
Yes, this is used to determine what code will be running on the gateway itself (so we can check for compatibility), so `none` made the most sense. Also, it'd be confusing to allow both `nodejs` and `javascript` but have them mean different things. `nodejs` should probably become `javascript`, honestly.

Reply:
OK